docs/scenarios/cli_demo.md

```diff
@@ -0,0 +1,203 @@
+
+
+> [!IMPORTANT]
+>
+> The demo assumes you are using IBM-served Llama 3. If you are using a different
+> model, the responses you get back may be different.
+
+### Running Kai with IBM-served Llama 3
+
+<!-- Begin copy from llm_selection.md#ibm-bam-service -->
+
+> [!WARNING]  
+> In order to use this service an individual needs to obtain a w3id
+> from IBM. The kai development team is unable to help obtaining this access.
+
+1. Login to https://bam.res.ibm.com/.
+2. To access via an API you can look at ‘Documentation’ after logging into
+   https://bam.res.ibm.com/. You will see a field embedded in the
+   'Documentation' section where you can generate/obtain an API Key.
+3. Ensure you have exported the key via `export
+GENAI_KEY=my-secret-api-key-value`.
+
+<!-- End copy from llm_selection.md#ibm-bam-service -->
+
+Next, paste the following into your `example/config.toml` file:
+
+```toml
+[models]
+provider = "ChatIBMGenAI"
+
+[models.args]
+model_id = "meta-llama/llama-3-70b-instruct"
+parameters.max_new_tokens = 2048
+```
+
+### Running Kai with Amazon Bedrock-served Llama 3
+
+1. Obtain your AWS API key from Amazon Bedrock.
+2. Export your `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and
+   `AWS_DEFAULT_REGION` environment variables.
+
+Next, paste the following into your `build/config.toml` file:
+
+```toml
+[models]
+provider = "ChatBedrock"
+
+  [models.args]
+  model_id = "meta.llama3-70b-instruct-v1:0"
+```
+
+Finally, run the backend with `podman compose up`.
+
+### Running Kai with GPT-3.5-Turbo
+
+<!-- Begin copy from llm_selection.md#openai-service -->
+
+1. Follow the directions from OpenAI
+   [here](https://help.openai.com/en/articles/4936850-where-do-i-find-my-openai-api-key).
+2. Ensure you have exported the key via `export
+OPENAI_API_KEY=my-secret-api-key-value`
+
+<!-- End copy from llm_selection.md#openai-service -->
+
+Next, paste the following into your `example/config.toml` file:
+
+```toml
+[models]
+provider = "ChatOpenAI"
+
+[models.args]
+model = "gpt-3.5-turbo"
+```
+
+### Running Kai with Cached Responses Only (demo mode)
+
+By default, this example demo script will run Kai in demo mode, which uses
+cached responses.  This is helpful if you don't have access to an LLM.
+Additionally, the responses are determinstic rather than slight changes based
+oan LLM response.
+
+### Install Kai
+Download Kai server and analyzer binaries for your machine and place them in
+our example directory. First, download the zip file for your OS
+[here](https://github.com/konveyor/kai/releases/tag/v0.0.1). Unzip the
+directory and copy the binaries to `example/analysis`.
+```
+$ cp ~/Downloads/kai-rpc-server.linux-x86_64/kai-rpc-server example/analysis/
+$ cp ~/Downloads/kai-rpc-server.linux-x86_64/kai-analyzer-rpc example/analysis/
+```
+
+### Setup analysis dependencies
+
+#### Start JDTLS in docker
+The simplest way to start JDTLS is to use docker:
+```
+$ docker run -d --name=bundle quay.io/konveyor/jdtls-server-base:latest
```

Comment:
I don't think you need this step

Reply:
ahh well, you need the container running, lets just make sure that we say that we need it running to copy out of the container and then once we copy all the things lets remove it